fitbenchmarking/fitbenchmarking
# Enable Azure to run on most unit tests

#### Description of Work

Enable Azure to run on most unit tests.

Enabled on all tests except

fitbenchmarking\controllers\test\test_externals_controllers.py
fitbenchmarking\cli\tests\test_main.py (scheduled to be removed or heavily modified in other issue)

Fixes #511

#### Testing Instructions

1. check code
2. check tests parse

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
